pkgs/top-level/aliases.nix

```
@@ -824,6 +824,7 @@ mapAliases {
   emacsNativeComp = emacs; # Added 2022-06-08
   emacsWithPackages = throw "'emacsWithPackages' has been renamed to/replaced by 'emacs.pkgs.withPackages'"; # Converted to throw 2024-10-17
   emacsPackages = emacs.pkgs; # Added 2025-03-02
+  emanote = throw "'emanote' has been removed due to lack of a maintainer"; # Added 2025-09-18
```

```suggestion
  emanote = throw "'emanote' has been removed due to lack of a Nixpkgs maintainer"; # Added 2025-09-18
```

I know it's somewhat implied, but let's be unambiguous, as upstream is maintained.